kubernetes-itests/src/test/java/io/fabric8/kubernetes/ConfigMapIT.java

```diff
@@ -0,0 +1,112 @@
+
+import static junit.framework.TestCase.assertTrue;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(ArquillianConditionalRunner.class)
+@RequiresKubernetes
+public class ConfigMapIT {
+  @ArquillianResource
+  KubernetesClient client;
+
+  @ArquillianResource
+  Session session;
+
+  private ConfigMap configMap1, configMap2;
+
+  private String currentNamespace;
+
+  @Before
+  public void init() {
+    currentNamespace = session.getNamespace();
+    configMap1 = new ConfigMapBuilder()
+      .withNewMetadata().withName("configmap1").endMetadata()
+      .addToData("1", "one")
+      .addToData("2", "two")
+      .addToData("3", "three")
+      .build();
+    configMap2 = new ConfigMapBuilder()
+      .withNewMetadata().withName("configmap2").endMetadata()
+      .addToData("PostgreSQL", "Free Open Source Enterprise Database")
+      .addToData("DB2", "Enterprise Database , It's expensive")
+      .addToData("Oracle", "Enterprise Database , It's expensive")
+      .addToData("MySQL", "Free Open SourceDatabase")
+      .build();
+    client.configMaps().inNamespace(currentNamespace).create(configMap1);
+    client.configMaps().inNamespace(currentNamespace).create(configMap2);
+  }
+
+  @Test
+  public void create() {
+    ConfigMap aConfigMap = client.configMaps().inNamespace(currentNamespace).load(getClass().getResourceAsStream("/test-configmap.yml")).get();
+    assertThat(aConfigMap).isNotNull();
+    assertEquals("game-config", aConfigMap.getMetadata().getName());
+  }
+
+  @Test
+  public void get() {
+    configMap1 = client.configMaps().inNamespace(currentNamespace).withName("configmap1").get();
+    assertThat(configMap1).isNotNull();
+    configMap2 = client.configMaps().inNamespace(currentNamespace).withName("configmap2").get();
+    assertThat(configMap2).isNotNull();
+  }
+
+  @Test
+  public void list() {
+    ConfigMapList aConfigMapList = client.configMaps().inNamespace(currentNamespace).list();
+    assertNotNull(aConfigMapList);
+    assertEquals(2, aConfigMapList.getItems().size());
+  }
+
+  @Test
+  public void update() {
+    configMap1 = client.configMaps().inNamespace(currentNamespace).withName("configmap1").edit()
+      .addToData("4", "four").done();
+    configMap2 = client.configMaps().inNamespace(currentNamespace).withName("configmap2").edit()
+      .addToData("MSSQL", "Microsoft Database").done();
+    assertNotNull(configMap1);
+    assertNotNull(configMap2);
+    assertEquals("four", configMap1.getData().get("4"));
+    assertEquals("Microsoft Database", configMap2.getData().get("MSSQL"));
+  }
+
+  @Test
+  public void delete() {
+    assertTrue(client.configMaps().inNamespace(currentNamespace).withName("configmap1").delete());
+    assertTrue(client.configMaps().inNamespace(currentNamespace).withName("configmap2").delete());
+  }
+
+  @After
+  public void cleanup() {
```

Comment:
Wondering if Arquilian cube provides any cleanup method since it is coming with own test runner.